spec/support/feature_helpers.rb

```diff
@@ -326,6 +328,41 @@ def form_is_filled_in_by_form_filler(live_form_link, skip_question: false, confi
     end
   end
 
+  def s3_form_is_filled_in_by_form_filler()
+    runner_url =  ENV.fetch('FORMS_RUNNER_URL') { raise 'You must set $RUNNER_URL' }
+    form_id =  ENV.fetch('S3_FORM_ID') { raise 'You must set $S3_FORM_URL' }
+    s3_form_live_link = runner_url + '/form/' + form_id
+
+    logger.info
+    logger.info "As a form filler"
+
+    logger.info "When I fill out the new form"
+    visit s3_form_live_link
+
+    logger.info "And I answer all of the questions"
+    answer_single_line(answer_text)
+
+    logger.info "Then I can check my answers before I submit them"
+    expect(page).to have_content 'Check your answers before submitting your form'
+    expect(page).to have_content answer_text
+
+    choose "No", visible: false
+    click_button 'Submit'
+
+    expect(page).to have_content 'Your form has been submitted'
+    reference_number = page.find("h1").sibling('div').text[-8..-1]
```

Comment:
It might be worth attaching an `id` to the HTML tag containing the reference number in forms-runner, so we don't need to rely on CSS selectors; I'd worry about changes to the page layout accidentally breaking this test.